src/imageio/format/avif.c

```diff
@@ -135,6 +135,14 @@ static int flp2(int i)
 
 void init(dt_imageio_module_format_t *self)
 {
+  if (codecName == NULL)
+  {
+    dt_print(DT_DEBUG_IMAGEIO,
+             "libavif doesn't offer encoding support!\n");
+    self->ready = FALSE;
+    return;
+  }
```

Comment:
I'm a bit pissed at CI - this change caused windows "no-features" build to fail due to "undefined `codecName` see eg https://github.com/johnny-bit/darktable/runs/1585302097?check_suite_focus=true#step:4:7031

Reply:
How did this even compile at all. There  is a line missing

Reply:
Yes, a line missing and it compiles on my side because I don't have libavif :(

Reply:
https://github.com/darktable-org/darktable/pull/7375